TrafficCapture/KafkaPrinter/src/main/java/org/opensearch/KafkaPrinter.java

```diff
@@ -58,6 +58,11 @@ public static void main(String[] args) {
         properties.setProperty("value.deserializer", "org.apache.kafka.common.serialization.ByteArrayDeserializer");
         properties.setProperty(ConsumerConfig.GROUP_ID_CONFIG, groupId);
         properties.setProperty(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
+        // Required for using SASL auth with MSK public endpoint
+        //properties.setProperty("security.protocol", "SASL_SSL");
+        //properties.setProperty("sasl.mechanism", "AWS_MSK_IAM");
+        //properties.setProperty("sasl.jaas.config", "software.amazon.msk.auth.iam.IAMLoginModule required;");
+        //properties.setProperty("sasl.client.callback.handler.class", "software.amazon.msk.auth.iam.IAMClientCallbackHandler");
```

Comment:
I guess this is the zero-latency fix that you're going to uncomment after Greg's E2E PR is merged?

Reply:
This is required properties to be able to communicate with an MSK public endpoint. By default I want to allow all kafka clusters but want to keep settings to be able to communicate with MSK if needed

Reply:
Oh I see, I misunderstood the description of the PR which made me get to this assumption. I apologize.

Reply:
Yes ideally I would like to have a config file here